docs/DATABASE.md

```diff
@@ -0,0 +1,32 @@
+Plan is to update the database scheme in a way that it will support the requirements we have as for the API specs and additional improvements of the deamon.
+
+## Tables
+### Orders (`orders`)
+- order - String: order identifier provided by the frontend 
+- payment_status - Enum: (pending|paid|timed_out). 
+- withdrawal_status - Enum: (waiting|failed|completed|none). 
```

Comment:
we might need "forced" status as well

Reply:
Addressed in [c69aa99](https://github.com/Alzymologist/Kalatori-backend/pull/80/commits/c69aa99b6747242f1a30f2348185c35d147486d8)